src/pages/[...slug].tsx

```diff
@@ -267,37 +277,29 @@ const ContentPage: NextPage<ContentPiece> = ({
             >
                 <GridLayout
                     sx={{
-                        rowGap: ['inc90', null, null, 'inc130'],
+                        rowGap: 0,
                     }}
                 >
                     <div sx={sideNavStyles}>
                         <SideNav currentUrl="#" items={tertiaryNavItems} />
                     </div>
 
-                    <div
-                        sx={{
-                            gridColumn: [
-                                'span 6',
-                                null,
-                                'span 8',
-                                'span 12',
-                                '4 /span 6',
-                            ],
-                        }}
-                    >
-                        {contentHeader}
-                        {contentBody}
-                        {contentFooter}
-                    </div>
+                    {contentHeader}
+                    {contentBody}
+                    {contentFooter}
                     {!vidOrPod && (
                         <div
                             sx={{
                                 display: ['none', null, null, null, 'block'],
                                 gridColumn: '10 /span 3',
+                                gridRow: '2 /4',
                             }}
                         >
                             {headingNodes.length > 0 && (
-                                <TableOfContents headingNodes={headingNodes} />
+                                <TableOfContents
+                                    headingNodes={headingNodes}
+                                    sx={{ position: 'sticky', top: 'inc150' }}
```

Comment:
what is the purpose of this top?

Reply:
So that is to push the table of contents below the top nav when we scroll down. If we don't have this the nav will cover it up.